src/Parser/Shaman/Elemental/Modules/Talents/ElementalBlast.js

```diff
@@ -14,6 +14,26 @@ class ElementalBlast extends Analyzer {
     combatants: Combatants,
   };
 
+  currentBuffAmount=0;
+  lastFreshApply=0;
+  resultDuration=0;
+
+  on_removebuff(event) {
```

Comment:
Would be better to prefix it also with `toPlayer` - `on_toPlayer_removebuff`. These would fire only when the target is the player. Same with `applybuff`.